CLI.md

```diff
@@ -268,6 +268,14 @@ $ zanshin alert list [OPTIONS] ORGANIZATION_ID
 * `--scan-target-id UUID`: Only list alerts from the specifiedscan targets.
 * `--states [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, ACTIVE, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
 * `--severity [CRITICAL|HIGH|MEDIUM|LOW|INFO]`: Only list alerts with the specifiedseverities  [default: CRITICAL, HIGH, MEDIUM, LOW, INFO]
+* `--language [pt-BR|en-US]`: Show alert titles in the specified language  [default: en-US]
```

Comment:
Ah, my bad, forgot to update this. I should add a PR template for this repo too.

Reply:
The CI/CD does re-generate it before uploading to PyPI... maybe we could change it to commit to `main` as well if there are any differences?